pkg/executor/default.go

```diff
@@ -74,24 +75,23 @@ func (l opList) uniqueNames() {
 	}
 }
 
-func (e *DefaultExecutor) applyStage(stage schema.Stage, fs vfs.FS, console plugins.Console) error {
+func (e *DefaultExecutor) applyStage(stageName string, stage schema.Stage, fs vfs.FS, console plugins.Console) error {
 	var errs error
 	for _, p := range e.conditionals {
 		if err := p(e.logger, stage, fs, console); err != nil {
 			e.logger.Warnf("(conditional) Skip '%s' stage name: %s",
-				err.Error(), stage.Name)
+				err.Error(), stageName)
 			return nil
 		}
 	}
-
 	e.logger.Infof(
 		"Processing stage step '%s'. ( commands: %d, files: %d, ... )",
-		stage.Name,
+		stageName,
 		len(stage.Commands),
 		len(stage.Files))
 
-	b, _ := json.Marshal(stage)
-	e.logger.Debugf("Stage: %s", string(b))
+	litter.Config.HideZeroValues = true
+	e.logger.Debugf("Stage: %s", litter.Sdump(stage))
```

Comment:
this dumps the actual stages in a much more nicer way. It shows no empty values, so only things that are gonna be run and its in a nice structured way which makes it simpler to understand visually.